sysdeps/linux/generic/sysdeps.cpp

```diff
@@ -893,6 +893,24 @@ int sys_ttyname(int fd, char *buf, size_t size) {
 	return 0;
 }
 
+int sys_pause() {
+#ifdef SYS_pause
+	auto ret = do_syscall(SYS_pause);
+#else
+	auto ret = do_syscall(SYS_ppoll, 0, 0, 0, 0);
+#endif
```

Comment:
Why?

Reply:
Presumably RISC-V does not have a pause syscall.

Reply:
Indeed, RISC-V does not have pause.